app/presenters/generic_file_presenter.rb

```diff
@@ -23,4 +24,8 @@ class GenericFilePresenter < Sufia::GenericFilePresenter
   delegate :production_names, :venue_names, :venue_full_names, :event_type_name, to: :model
 
   delegate :public?, :discoverable?, :restricted?, :private?, to: :model
+
+  def has_year_only
```

Comment:
This is a form field, so the method name has to match, right?  If I'm wrong about that, I'd be happy to make this change.